src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As already explained above, the value of the action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)